routes/renderers.js

```diff
@@ -85,12 +85,12 @@ function _renderDashboard(req, res, next, opts) {
         .then(function success(notebook) {
             debug('Success loading nb');
 
-            // Check if some cell in the notebook has the dashboard metadata
-            var hasDashboardLayout = notebook.cells.some(function(currentCell){
-                return ( currentCell.hasOwnProperty('metadata') &&
-                         currentCell.metadata.hasOwnProperty('urth') &&
-                         currentCell.metadata.urth.hasOwnProperty('dashboard') );
-            });
+            // get dashboard layout from notebook-level dashboard metadata
+            var dashboardLayout = (notebook.metadata &&
+                                  notebook.metadata.urth &&
+                                  notebook.metadata.urth.dashboard &&
+                                  notebook.metadata.urth.dashboard.layout) ||
+                                  'no-layout';
```

Comment:
I agree, and this is what @jameslmartin did in #145. I can reinstate that check.